Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.6.3
+
+ENV SHELL="/bin/bash"
+
+RUN apt-get update && \
+    apt-get install -y git \
+                       wget \
+                       python3-pip \
+                       vim
+
+RUN wget https://github.com/dssg/obscuritext/archive/develop.tar.gz && \
```

Comment:
use the SHA instead. You can find it here: https://github.com/dssg/obscuritext/commits/develop